src/adapters/curve/products/staking/curveStakingAdapter.ts

```diff
@@ -216,6 +225,7 @@ export class CurveStakingAdapter
   }
 
   @ResolveUnderlyingMovements
+  @AddClaimedRewards({ rewardAdapterIds: ['reward'] })
```

Comment:
we resolve the underlying first then add claimed rewards

Reply:
I don't think there's any way to enforce the order via typescript, so we'll have to keep an eye in future cases.

Reply:
good point i wonder if the order matters, might actually get the same results either way 🤔